### DISCUSSION TITLE
Table Editor: Grid fails to load cells if cell value is extremely large

### DISCUSSION THREAD

**Discussion by joshenlim** [supabase/supabase#16931] [Contribute to Supabase]
We're currently actively looking into this - creating an issue here to track progress
Can reproduce by creating a table with a jsonb column, then inserting a row with a really large JSON

**Comment by Bhavyawahie** [comment]
I tried replicating the same on Supabase platform and the application froze for a good few minutes displaying nothing but a white screen and consequently yielded into the following error.
JSON blob used: https://github.com/json-iterator/test-data/blob/master/large-file.json

>![image](https://github.com/supabase/supabase/assets/34278282/9bf9c001-559b-4028-b386-2667eb389c6a)
